cmd/spectrum-x-manager/main.go

```diff
@@ -170,6 +170,15 @@ func main() {
 		os.Exit(1)
 	}
 
```

Comment:
Let's have a flag for the SRIOV Object namespace and pass it to controller:
```
flag.StringVar(&sriovObjNamespace, "sriov-obj-namespace", "default", "SRIOV Network Operator namespace")
```

Also add it to Deployment here:
https://github.com/Mellanox/spectrum-x-operator/blob/main/config/manager/manager.yaml#L54